dotnet/aspnetcore-runtime-current/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                aspnetcore-runtime-current
```

Comment:
Instead of "current", it's better to specify a version, eg 50.

Reply:
here the motivation is to signal that this is a 'rolling' release, that is *not* too stable and should not be used for anything other that experimenting. whenever 6.0 is released, current will switch to next 'odd' major *current* release, that is 7.x. If I name it *-5, it will become obsolete too quickly (3 months after 6.0 release, that is coming this year)

Reply:
May be it is better to also explicitly state that in port description

Reply:
If it's "not too stable" then the correct suffix in MacPorts would probably be "-devel".